blackboxopt/visualizations/utils.py

```diff
@@ -13,7 +13,7 @@
 from blackboxopt import Evaluation, Objective
 
 # For backwards compatibility; TODO: remove with next major release
-from blackboxopt.utils import mask_pareto_efficient
+from blackboxopt.utils import mask_pareto_efficient  # noqa
```

Comment:
Personally, I prefer to ignore only specific rules, e.g. `# noqa: ANN201` or something. Some benefits I see:
- Not (accidentally) masking other errors
- Helps human reader to understand the ignore without the need to run `ruff` (e.g. in PR reviews)
- It helps `ruff` (e.g. if a rule got deprecated, ruff will warn/auto-remove the `noqa`'s for that specific rule)